docs/setup/install-existing.md

```diff
@@ -55,13 +55,15 @@ To install AAD Pod Identity to your cluster:
    - *RBAC enabled* AKS cluster
 
   ```bash
-  kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.6.0/deploy/infra/deployment-rbac.yaml
+  kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.7.0/deploy/infra/deployment-rbac.yaml
+  kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.7.0/deploy/infra/mic-exception.yaml
```

Comment:
is mic-exception.yaml mandatory only for AAD Pod Identity Add-on?

Reply:
this guide is for helm-brownfield. I didn't touch add-on version ins this PR, because it's in a preview and preview is not relevant for my project

Reply:
I've picked aad part of the guide from here when tested on my project https://github.com/Azure/aad-pod-identity/releases/tag/v1.7.0